src/initializers/riviere.ts

```diff
@@ -22,7 +22,11 @@ const init = (config, orkaOptions) => {
       https: true,
       level: 'info',
       maxBodyValueChars: config.riviere.maxBodyValueChars,
-      blacklistedPathRegex: config.riviere.outbound && config.riviere.outbound.blacklistedPathRegex
+      blacklistedPathRegex: config.riviere.outbound && config.riviere.outbound.blacklistedPathRegex,
+      request: {
+        enabled: config.riviere.outbound.request.enabled
```

Comment:
❓ Is there a default value for `config.riviere.outbound.request.enabled`  in diamorphosis?

Reply:
LOL! You were fast

Reply:
😄